internal/dinosql/gen.go

```diff
@@ -1145,6 +1145,14 @@ const (
 	{{- end}}
 )
 
+{{ if $.EmitEnumMapping }}
+var Every{{.Name}} = map[{{.Name}}]bool{
```

Comment:
Looking over this, don't you want these to be the string values of the enums? Not the Enum values themselves?

Reply:
Either way... can just as easily do e.g `if db.EveryCompanyRole[CompanyRole("unknownstring")]`

Reply:
I'll leave it up to you 👍